docs/javascript/ionic-core.md

```diff
@@ -0,0 +1,110 @@
+---
+title: 'Ionic Core Custom Components'
+sidebar_label: Ionic Core
```

Comment:
Let's rename it to Quickstart to match the other frameworks. The filename should also be changed.

```suggestion
title: 'Intro Ionic JavaScript Quickstart'
sidebar_label: Quickstart
```